thegreatknowledgeheist/bert/bert.py

```diff
@@ -0,0 +1,54 @@
+from transformers import (
```

Comment:
Ja bym chyba przeniósł katalog bert do models i ewentualnie zmienił jego nazwę na base_bert czy coś takiego, no ale to kwestia preferencji.